hardhat/docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+version: '3.8'
+
+name: lilypad-chain
+
+services:
+  #lilypad-node:
+  #  build:
+  #    context: infra/ava
+  #    dockerfile: Dockerfile
+  #  entrypoint: tail -f /dev/null
+  #  ports:
+  #    - 9650:9650
+  #    - 9652:9652
+  #    - 9654:9654
+  #    - 9656:9656
+  #    - 9658:9658
+  
+  lilypad-geth:
+    build:
+      context: infra/geth
+      dockerfile: Dockerfile
+    ports:
```

Comment:
What's the purpose for port `30303`?

Reply:
https://ethereum.stackexchange.com/questions/809/which-tcp-and-udp-ports-are-required-to-run-an-ethereum-client

Reply:
I am mostly asking to get context/learn about this, do we need to expose the 3 ports (`8545`, `8546` and `30303`) if we run a node for testnet? Is it expected and good practice to do so?